src/pcre2_jit_compile.c

```diff
@@ -3302,7 +3302,7 @@ OP2(SLJIT_SUB | SLJIT_SET_Z, COUNT_MATCH, 0, COUNT_MATCH, 0, SLJIT_IMM, 1);
 add_jump(compiler, &common->calllimit, JUMP(SLJIT_ZERO));
 }
 
-static SLJIT_INLINE void allocate_stack(compiler_common *common, int size)
+static SLJIT_INLINE void allocate_stack(compiler_common *common, sljit_uw size)
```

Comment:
probably should had been PCRE2_SIZE but were missed when PCRE2 was created from PCRE1 (which used `int` for all offsets)

Reply:
`PCRE2_SIZE` substituted, thanks!